public/app/screens/main/sections/device-details/style.scss

```diff
@@ -46,46 +49,33 @@ $column-factor: $total-columns / $content-columns;
   &__controls,
   &__properties,
   &__access-rights,
-  &__details{
+  &__details {
     @include grid-row();
```

Comment:
We still have everything as-is in these PR. 
We will change all things in a future PR and we'll see what will be changed